extensions/blocks/videopress/seekbar-color-settings.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { PanelColorSettings } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+const SeekbarColorSettings = ( { attributes, setAttributes } ) => {
+	const handleChangeSeekbarColor = seekbarColor => setAttributes( { seekbarColor } );
+
+	const handleChangeSeekbarLoadingColor = seekbarLoadingColor =>
+		setAttributes( { seekbarLoadingColor } );
+
+	const handleChangeSeekbarPlayedColor = seekbarPlayedColor =>
+		setAttributes( { seekbarPlayedColor } );
+
+	return (
+		<PanelColorSettings
+			title={ __( 'Seekbar Colors' ) }
```

Comment:
We'll need to add the textdomain here and below.

```suggestion
			title={ __( 'Seekbar Colors', 'jetpack' ) }
```